code/modules/surgery/organs/organ.dm

```diff
@@ -32,6 +32,13 @@ var/list/organ_cache = list()
 	var/sterile = 0 //can the organ be infected by germs?
 	var/tough = 0 //can organ be easily damaged?
 
+/obj/item/organ/Destroy()
+	processing_objects.Remove(src)
+	owner = null
+	QDEL_LIST_ASSOC_VAL(autopsy_data)
```

Comment:
I honestly don't know why autopsy data is on all organs when it can only ever apply to external (trace chemicals are another story).